psi4/src/psi4/detci/ciwave.cc

```diff
@@ -51,19 +51,13 @@ PRAGMA_WARNING_POP
 namespace psi {
 namespace detci {
 
-CIWavefunction::CIWavefunction(std::shared_ptr<Wavefunction> ref_wfn) : Wavefunction(Process::environment.options) {
-    // Copy the wavefuntion then update
-    shallow_copy(ref_wfn);
-    set_reference_wavefunction(ref_wfn);
+CIWavefunction::CIWavefunction(std::shared_ptr<Wavefunction> ref_wfn, Options& options)
+    : Wavefunction{ref_wfn, options} {
     common_init();
 }
 
-CIWavefunction::CIWavefunction(std::shared_ptr<Wavefunction> ref_wfn, Options& options) : Wavefunction(options) {
-    // Copy the wavefuntion then update
-    shallow_copy(ref_wfn);
-    set_reference_wavefunction(ref_wfn);
-    common_init();
```

Comment:
where'd `common_init()` go?

Reply:
This constructor delegates to the two-argument one above, which calls `common_init`.